clouddriver-alicloud/src/main/java/com/netflix/spinnaker/clouddriver/alicloud/AliCloudConfigurationProperties.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2019 Alibaba Group.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.clouddriver.alicloud;
+
+/**
+ * @author: luoguan
```

Comment:
for future commits can you omit this javadoc? 

we don't tend to add `@author` tags in the codebase, and `@create` is not valid javadoc

Reply:
after #3892 invalid javadoc will fail the PR build

Reply:
Hi @cfieber I am sorry for that. I will remove them in the next PRs. Thanks for your suggestion.